agr_java_core/src/main/java/org/alliancegenome/neo4j/view/View.java

```diff
@@ -1,9 +1,11 @@
 package org.alliancegenome.neo4j.view;
 
+import org.alliancegenome.curation_api.view.View.ForPublic;
+
 public class View {
 
 	// Generic Views
-	public static class Default {
+	public static class Default extends ForPublic {
```

Comment:
Was there something that didn't work due to this? ... Also in the future we might want to get rid of this View file and just use the Curation one... just a thought.